### PR TITLE
finished game logic

### PR DESCRIPTION
**Pull request by Ted-Smiles**
Added a found group state that contents all the found item  

Added a different colour for each found group

Added some styling to the button and disabled it when there wasn't a valid guess

Added a play again option that for now only empty the found groups

Changed some styling on the front page

### CODE REVIEW

File: frontend/src/components/Game.tsx [hunk 1]
```diff
@@ -91,13 +107,31 @@ export default function Game() {
 
     for (let i = 0; i < groups.length; i++) {
       if (groups[i].sort().join('') === guess) {
-        console.log('winner');
+        setFoundGroups([...foundGroups, ...selected]);
         setGuessResult('correct');
+
+        console.log(foundGroups.length);
```
Comment (tartebabba) [inline frontend/src/components/Game.tsx:113]: is this just so we can see feedback for the time being?

Comment (Ted-Smiles) [reply frontend/src/components/Game.tsx:113]: Not really need tbh

File: frontend/src/components/Game.tsx [hunk 2]
```diff
@@ -123,16 +165,28 @@ export default function Game() {
...
+        <TouchableOpacity
+          disabled={selected.length !== 4}
+          style={selected.length !== 4 ? styles.disableButton : styles.button}
+          onPress={handleSubmit}
+        >
           <Text>Submit</Text>
         </TouchableOpacity>
         {guessResult !== '' ? (
-          guessResult === 'correct' ? (
+          guessResult === 'correct' || guessResult === 'winner' ? (
```
Comment (tartebabba) [inline frontend/src/components/Game.tsx:176]: All good - interested in the '' default state

**Review by tartebabba — APPROVED** [review]
Looks good

Are you guys just using the basic types for typescript or will be specifying the specific types later?

**Pull request merged**